### PR TITLE
Add two line `do:` preference

### PR DESCRIPTION
**Pull request by bcardarella**
Thoughts?

### CODE REVIEW

File: engineering/elixir.md [hunk 1]
```diff
@@ -68,14 +70,14 @@ def send, do: # ...
 * ...but put function `def`s that match for the same function together.
 
 ```elixir
-def sum(nil), do: {:error, "No Value"}
+def sum(nil),
+  do: {:error, "No Value"}
 def sum([]), do: 0
```
Comment (danmcclain) [inline engineering/elixir.md]: What about this one?

Comment (danmcclain) [inline engineering/elixir.md]: Ah, nevermind, saw the wording below

**Comment by danmcclain** [comment]
I am :+1: on this

File: engineering/elixir.md [hunk 2]
```diff
@@ -127,6 +129,24 @@ end
 def iterate([]), do: :ok
 ```
 
+* Favor `do:` on its own line for singline lines expressed in more than a single word
```
Comment (brzpegasus) [inline engineering/elixir.md]: `s/singline/single/`

**Comment by brzpegasus** [comment]
Works for me.

**Pull request merged**